src/nupic/utils/CSVHelpers.hpp

```diff
@@ -0,0 +1,139 @@
+#ifndef NTA_UTILS_CSVHELPERS
+#define NTA_UTILS_CSVHELPERS
+
+#include <iostream>
+#include <boost/tokenizer.hpp>
+#include <vector>
+#include <string>
+#include <memory>
+#include <fstream>
+
+#include <nupic/types/Types.hpp>
+//#include "nupic/utils/VectorHelpers.hpp"
+
+namespace nupic {
+namespace utils {
+/** 
+* defines I/O classes for CSV - CSVReader & CSVWriter
+*/
+namespace csv {
+
+//using namespace nupic::utils;
+// read
+template<class T>
```

Comment:
should we use the template here? or let `getLine()` and `readColumn()` return vector of strings (and use vector utils castVectorType() if needed)?